stdlib/public/TensorFlow/TensorHandle.swift

```diff
@@ -44,6 +42,20 @@ public final class TensorHandle<Scalar : AccelerableByTensorFlow> {
     self.cTensorHandle = cTensorHandle
   }
 
+  deinit {
+    debugLog("De-initializing TensorHandle.")
```

Comment:
Actually I'm going to move initializers back into `TensorHandle<T>` since only that would be correct, and `_AnyTensorHandle` is just a type-erased class for easier IRGen. So it's always a TensorHandle.